T7Suite/MapViewerEx.cs

```diff
@@ -3,24 +3,19 @@
 using System.ComponentModel;
 using System.Drawing;
 using System.Data;
-using System.Text;
 using System.Windows.Forms;
 using DevExpress.XtraEditors;
 using DevExpress.XtraCharts;
-using System.Runtime.InteropServices;
 using Nevron.Chart;
 using Nevron.GraphicsCore;
 using Nevron.Chart.WinForm;
 using DevExpress.XtraGrid.Views.Grid.ViewInfo;
-using DevExpress.XtraBars.Docking;
-using System.IO;
 using CommonSuite;
 using NLog;
 
 namespace T7
 {
 
-
     public partial class MapViewerEx : /*DevExpress.XtraEditors.XtraUserControl */IMapViewer
```

Comment:
This one is good to keep commented, as I need to toggle it back to view it in the designer.